operator/controllers/instance_controller.go

```diff
@@ -113,15 +133,15 @@ func (r *InstanceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	if instance.HelmValuesRef.Name == "" {
-		instance.HelmValuesRef.Name = fmt.Sprintf("%s-values", releaseReq.ReleaseName())
+		instance.HelmValuesRef.Name = fmt.Sprintf("%s-values-yaml", releaseReq.ReleaseName())
 		instanceNeedsUpdate = true
 	}
 
 	if instanceNeedsUpdate {
 		if err := r.Update(ctx, instance); err != nil {
 			return ctrl.Result{}, err
 		}
-		return ctrl.Result{RequeueAfter: 5 * time.Second}, nil
+		return ctrl.Result{}, nil
```

Comment:
No more requeue?

Reply:
No - I found out that the `Update` request triggers another event, and the reconcile loop gets triggered again.